### DISCUSSION TITLE
"trusted_proxies" setting is not correctly set

### DISCUSSION THREAD

**Discussion by ozboss** [nextcloud/all-in-one#5224] [Questions] [answered]
### Steps to reproduce
Create manual-install instance of nextcloud-aio.

### Expected behavior 
No errors in admin overview.

### Actual behavior 
Error in admin overview:
Your "trusted_proxies" setting is not correctly set, it should be an array of IP addresses - optionally with range in CIDR notation.

### Other information
The entrypoint.sh script of the nextcloud container automatically creates a trusted_proxies entry:
```
IPv4_ADDRESS="$(dig nextcloud-aio-nextcloud A +short +search | head -1)" 
IPv4_ADDRESS="$(echo "$IPv4_ADDRESS" | sed 's|[0-9]\+$|0/16|')" 
php /var/www/html/occ config:system:set trusted_proxies 10 --value="$IPv4_ADDRESS"
```

The problem is the result of the dig command is empty and the entry for trusted_proxies become an empty string.
The IP for nextcloud-aio-nextcloud is resolved by the entry in /etc/hosts that is created by docker and does not result in a dns lookup that dig can report on.

The error disappears from the admin overview after running this command:
`php occ config:system:delete trusted_proxies 10`

I am using podman but the behavior should be the same for docker.

**Comment by Zoey2936** [comment, marked as answer]
this problem should only exist in podman, which is not supported

**Comment by szaimen** [comment]
> this problem should only exist in podman, which is not supported

Indeed. 

See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-run-this-with-podman-instead-of-docker

**Comment by makinbacon21** [comment]
I have this issue with Docker as well, so...
> Your "trusted_proxies" setting is not correctly set, it should be an array of IP addresses - optionally with range in CIDR notation. For more details see the [documentation ↗](https://docs.nextcloud.com/server/30/go.php?to=admin-reverse-proxy).
